include/AudioSdl.h

```diff
@@ -39,7 +39,7 @@
 #include "AudioDevice.h"
 #include "AudioDeviceSetupWidget.h"
 
-class QLineEdit;
+class QComboBox;
```

Comment:
I am curious, is it possible to use the styled variant here?

Reply:
The settings dialog mainly uses the Qt combo boxes (as opposed to the LMMS combo boxes) for several reasons:
* The LMMS combo box is not HiDPI aware.
* IIRC the LMMS combo box needs a model so that it can automate the values. These are not available and do not make sense for entries in the dialog because none of the settings needs to be or can be automated.

The setup dialog has been switched to use mainly Qt combo boxes via pull request https://github.com/LMMS/lmms/pull/6779.